learn/advanced/known_limitations.md

```diff
@@ -57,6 +57,12 @@ If your query is `Hello - World`:
 
 **Explanation:** This limit is enforced for performance and storage reasons. Overly large internal data structures—resulting from documents with too many fields—lead to overly large databases on disk, and slower search performance.
 
+## Maximum number of concurrent requests
+
+**Limitation:** Meilisearch can receive a maximum of 126 API requests at the same time. Requests exceeding this limit will cause Meilisearch to thrown an internal error.
```

Comment:
```suggestion
**Limitation:** Meilisearch can receive a maximum of 126 API requests at the exact same time. Requests exceeding this limit will cause Meilisearch to thrown an internal error.
```

I would like to add this extra precision because I want people to fill this situation to be nearly impossible to trigger.